src/styles/blocks/star.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+
+  &-star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
```

Comment:
It's recommended to use relative paths for images. Use './images/star.svg' instead of '/images/star.svg'.